packages/react-devtools-shared/src/devtools/views/Components/InspectedElementPropsTree.js

```diff
@@ -88,7 +91,7 @@ export default function InspectedElementPropsTree({
             value={value}
           />
         ))}
-      {canEditValues && (
+      {!readOnly && canEditValues && (
```

Comment:
Can we make this part of the `canEditValues` boolean assignment?

Reply:
Done. I also introduced `canEditValuesProp` for consistency (see my comment below).